noria-server/src/integration.rs

```diff
@@ -2214,6 +2214,7 @@ fn test_queries(test: &str, file: &'static str, shard: bool, reuse: bool, log: b
             }
         }
     });
+    println!("{:#?}", g.graphviz());
```

Comment:
This was probably left in by accident?

Reply:
Er, yeah. That's how I was printing out a graph to check whether my optimization was actually doing anything on the real queries.